lib/libzfs_core/libzfs_core.c

```diff
@@ -664,9 +717,18 @@ lzc_send_resume_redacted(const char *snapname, const char *from, int fd,
 {
 	nvlist_t *args;
 	int err;
+	int pipefd[2];
+	pthread_t mythread;
+	pthread_attr_t myattr;
+	sendargs_t sendargs;
+	int dumbstatus;
+
+
+	err = pipe(pipefd);
+	err = pthread_attr_init(&myattr);
```

Comment:
myattr isn't used for anything

Reply:
Removed, thanks - didn't realize you could just call `pthread_create` with NULL instead.